build-logic/build-update-utils/src/main/kotlin/gradlebuild.update-versions.gradle.kts

```diff
@@ -5,6 +5,10 @@ import gradlebuild.buildutils.tasks.UpdateReleasedVersions
 import gradlebuild.buildutils.model.ReleasedVersion
 import java.net.URL
 
+plugins {
+    id("gradlebuild.module-identity")
```

Comment:
:x: This is causing issues. It's not visible on master right now, but on a branch I have.

This basically gives identity to the root build. The "utility" plugin should not do that. If we want/need to reuse the branch part we have to decouple it from the module identity extension.

Or we just do if without reusing code from somewhere else here. This should only provide utility tasks that are not called as part of regular build. There shouldn't be an issue if they have to "repeat work" done in another place.